client/src/app/+admin/config/edit-custom-config/edit-instance-information.component.html

```diff
@@ -126,6 +126,7 @@
 
           <my-markdown-textarea
             name="instanceTerms" formControlName="terms"
+            [customMarkdownRenderer]="getCustomMarkdownRenderer()" [debounceTime]="500"
```

Comment:
Just use `[markdownType]="enhanced"` instead

Reply:
✅ Thank you for the tip again!